flake.nix

```diff
@@ -33,12 +36,37 @@
           inherit system;
           overlays = [ overlay ];
         };
-      in
-      {
-        devShells.default = with pkgs; mkShell {
-          buildInputs = [
-            polykey-cli
-          ];
+      in rec {
+        devShells = {
```

Comment:
What was the reason to have `in rec`?

Reply:
I was referencing a package i defined for gitignore for testing, but following https://github.com/MatrixAI/nixpkgs-matrix/pull/4#issuecomment-2292635796, this is irrelevant now.